app/helpers/nodeHelper.js

```diff
@@ -9,6 +9,16 @@ angular.module('portainer.helpers')
         Labels: node.Spec.Labels,
         Availability: node.Spec.Availability
       };
+    },
+    getManagerIP: function(nodes) {
+      var manager_ip = undefined;
```

Comment:
Could you rename the variable `manager_ip` to `managerIP` to stay coherent with variable naming inside the project ?

Also, you do not need to assign it to `undefined`, so `var managerIP;` is just fine.